libs/design/src/molecules/backdrop/animation/backdrop-animation-state.ts

```diff
@@ -0,0 +1 @@
+export const getAnimationState = (interactable: boolean) => interactable ? 'interactable' : 'non-interactable';
```

Comment:
Is there a type out of `@angular/animations` that represents these return types? These hardcoded strings seem brittle.